### ISSUE TITLE
Database request

### ISSUE THREAD

**Issue by weizhou-geek** [yyxhpkq/MQNR#1]
Hello! Thanks for your great work! I wonder how to access the subjective scores and the fused images generated by different pansharpening methods?

**Comment by yyxhpkq** [comment]
&nbsp;Hello !&nbsp;I remember that there is a URL for the relevant database in 'A large-scale benchmark data set for evaluating pansharpening performance: overview and implementation. IEEE Geoscience and Remote Sensing Magazine'





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yyxhpkq/MQNR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月4日(星期五) 中午12:28
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[yyxhpkq/MQNR] Database request (Issue #1)





 
Hello! Thanks for your great work! I wonder how to access the subjective scores and the fused images generated by different pansharpening methods?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by weizhou-geek** [comment]
@yyxhpkq Yes. I’ve downloaded the data from that URL, but no subjective scores and fused results.

**Comment by yyxhpkq** [comment]
I'm sorry I don't have a database right now, you can find it in this article&nbsp; ’’A large-scale remote sensing database for subjective and objective quality assessment of pansharpened images. YiMing Xiong"


“”------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yyxhpkq/MQNR"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月5日(星期六) 下午3:23
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yyxhpkq/MQNR] Database request (Issue #1)





 
@yyxhpkq Yes. I’ve downloaded the data from that URL, but no subjective scores and fused results.
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by weizhou-geek** [comment]
I used the URL: https://pan.baidu.com/s/1Utja1PHUqgoXaksWjC2CPw (Password：y77y)

But there are no available subjective scores and fused images. Do you mean this link?
